qe_tools/parsers/_input_base.py

```diff
@@ -679,6 +694,36 @@ def _parse_atomic_species(txt):
     :raises qe_tools.utils.exceptions.ParsingError: if there are issues
         parsing the input.
     """
+    def validate_species_name(atom_name, pseudo_file_name):
```

Comment:
Please add a leading underscore to the method name, to indicate that this is not part of the _public_ interface of the class.

Reply:
done :+1: